org.adempiere.base/src/org/compiere/acct/Doc.java

```diff
@@ -2322,4 +2360,15 @@ protected MAcctSchema getAcctSchema() {
 	public boolean isDeferPosting() {
 		return false;
 	}
+	
+	/** Post Process **/
+	private boolean m_PostProcess;

```

Comment:
what exactly is post process means here ?

Reply:
True when the document is part of the post process. The System will re-post all the costing-relevance documents after a back-date document is posted.

Reply:
Renamed to isBackDateProcess